### PR TITLE
[no-release-notes] log $output when $status is a mismatch in merge tests

### PR DESCRIPTION
**Pull request by druvv**
Will revert this commit when it's clear what is causing the tests to flake.

### CODE REVIEW

File: integration-tests/bats/helper/common.bash [hunk 1]
```diff
@@ -78,6 +78,22 @@ teardown_common() {
     rm -rf "$BATS_TMPDIR/dolt-repo-$$"
 }
 
+status_eq() {
```
Comment (zachmu) [inline integration-tests/bats/helper/common.bash:81]: Call this log_status_eq

File: integration-tests/bats/helper/common.bash [hunk 2]
```diff
@@ -78,6 +78,22 @@ teardown_common() {
...
+    if ! [ "$status" -eq $1 ]; then
+        echo "status: expected $1, received $status"
+        printf "output:\n$output"
+        exit 1
+    fi
+}
+
+output_has() {
```
Comment (zachmu) [inline integration-tests/bats/helper/common.bash:89]: log_output_has

**Review by zachmu — COMMENTED** [review]
LGTM

Are the tests in constraint-violations.bats also flakey?

**Comment by druvv** [comment]
Yes I saw a flaky test in constraint-violation.bats since it uses merge.

**Pull request merged**